distance

```diff
@@ -1,3 +1,10 @@
+def dist x,y
```

Comment:
`def distance(start_point, end_point)`
Always use meaningful names. And use parenthesis. Check out this https://github.com/bbatsov/ruby-style-guide#method-parens 

Moreover, it looks like point can be extracted out as a class, what do you think?